### PR TITLE
Postgres 10 Sequence Support

### PR DESCRIPTION
**Pull request by jalissonmello**
Postgres 10 Sequence support

### CODE REVIEW

File: src/main/java/cz/startnet/utils/pgdiff/schema/PgConstraint.java [hunk 1]
```diff
@@ -56,7 +56,6 @@ public String getCreationSQL() {
         sbSQL.append(PgDiffUtils.getQuotedName(getTableName()));
         sbSQL.append(System.getProperty("line.separator"));
         sbSQL.append("\tADD CONSTRAINT ");
-        sbSQL.append(PgDiffUtils.getCreateIfNotExists());
```
Comment (chadfurman) [inline src/main/java/cz/startnet/utils/pgdiff/schema/PgConstraint.java]: why did we remove this line?

File: src/test/resources/cz/startnet/utils/pgdiff/add_constraint_diff.sql [hunk 2]
```diff
@@ -1,3 +1,3 @@
 
 ALTER TABLE testtable
-	ADD CONSTRAINT IF NOT EXISTS field4check CHECK (((field4 > (-5.0)::double precision) AND (field4 < (5.0)::double precision)));
```
Comment (chadfurman) [inline src/test/resources/cz/startnet/utils/pgdiff/add_constraint_diff.sql]: same here, what's wrong with the "if not exists" ?

**Review by chadfurman — DISMISSED** [review]
If someone could also really look over the SQL and pull-down / test this out, that would be helpful.  It's a bit different than the patch I saw earlier which was much simpler.

Also, what's up with removing the "if not exists" -- that seems un-related.

File: src/main/java/cz/startnet/utils/pgdiff/schema/PgConstraint.java (hunk 1 above)
Comment (jalissonmello) [reply src/main/java/cz/startnet/utils/pgdiff/schema/PgConstraint.java]: Add constraint doesn't support add if not exists, only drop if exists

Comment (jalissonmello) [reply src/main/java/cz/startnet/utils/pgdiff/schema/PgConstraint.java]: The constraint code is in the wrong PR, I will remove it because  there are the PR #226 with the same fix

**Comment by jalissonmello** [comment]
Now in the PR only Sequence related code.  I added test case to check create sequence on postgres 10.

File: src/test/resources/cz/startnet/utils/pgdiff/add_sequence_issue225_new.sql [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+SET default_with_oids = false;
+
+--
+-- Name: testtable; Type: TABLE; Schema: public; Owner: fordfrog; Tablespace: 
+--
+
+CREATE TABLE testtable (
+    field1 integer,
+    field2 integer,
```
Comment (steve-chavez) [inline src/test/resources/cz/startnet/utils/pgdiff/add_sequence_issue225_new.sql]: This table diffing is not related to this patch, the test would be clearer if it's not included. Only the `CREATE/ALTER SEQUENCE` are needed.

**Comment by steve-chavez** [comment]
Can confirm the patch works, the resulting SQL is good and `mvn test` is green.

**Comment by emmanueltouzery** [comment]
would be really nice if this gets merged :-) hit the problem myself. thanks for the work @steve-chavez :+1:

**Comment by GeertJohan** [comment]
Looks like this also solves @225?

**Comment by GeertJohan** [comment]
Looks like this also  solves #225 ?

**Comment by eseliger** [comment]
Anything we can do to push this forward? :)

**Comment by calexdiaz143** [comment]
Is there any more review that needs to be done or can this get pushed?

File: src/test/resources/cz/startnet/utils/pgdiff/add_constraint_diff.sql (hunk 2 above)
Comment (jalissonmello) [reply src/test/resources/cz/startnet/utils/pgdiff/add_constraint_diff.sql]: The constraint related code was removed. Only sequence code in the PR now

**Comment by valoricDe** [comment]
In order to get my fresh schema dump working with https://github.com/valorize/postgres-schema-updater I had to also merge #240 into develop.
As the tests are all green and the changes requested by @chadfurman have been made I'm going to be bold and will merge this into fordfrog:develop.
